lib/master.go

```diff
@@ -131,10 +132,16 @@ func NewBaseMaster(
 	var (
 		nodeID        p2p.NodeID
 		advertiseAddr string
+		masterMetaExt = &MasterMetaExt{}
 	)
 	if ctx != nil {
```

Comment:
just out of curiosity, when is it nil? I see it's always assigned when "create worker"

Reply:
maybe in mock module, such as https://github.com/hanfei1991/microcosm/blob/ee6a157c0c101730e6799c2973a863b861034a1b/lib/mock_master_impl.go#L60-L61